Sources/Datadog/Tracing/AutoInstrumentation/URLSessionTracingHandler.swift

```diff
@@ -68,6 +68,12 @@ internal class URLSessionTracingHandler: URLSessionInterceptionHandler {
                 }
             }
         }
+        let appStateHistory = interception.appStateListener.history.take(
+            between: resourceMetrics.fetch.start...resourceMetrics.fetch.end
+        )
+        // TODO: RUMM-1064 tag doesn't seem like the best fit but can't find anything better
+        span.setTag(key: "foreground_duration", value: "\(appStateHistory.foregroundDuration)")
+        span.setTag(key: "is_background", value: "\(appStateHistory.didRunInBackground)")
```

Comment:
1. should they be declared in `DDTags`? (unlike other `DDTags` these are not officially supported by Datadog)
2. should they be added as `tag`s? (i couldn't think of any other place for these)
cc @ncreated

Reply:
i removed string conversion, now it's
```swift
span.setTag(key: "foreground_duration", value: appStateHistory.foregroundDuration.toNanoseconds)
```
with this, you can create a `measure` (open the trace, click on `foreground_duration` tag, select add facet, select `measure` tab at the top) and filter traces longer/shorter than your threshold

NOTE: once you do that, it takes some time (a few minutes maybe) before you can use it in your query. at first i thought it wasn't working but it actually works 👌 

<img width="1317" alt="example-query" src="https://user-images.githubusercontent.com/1417500/112363044-60823200-8cd5-11eb-8200-83863320136f.png">